README.md

```diff
@@ -16,3 +16,12 @@ npm start
 yarn install
 yarn start
 ```
+
+**Docker setup**
+```shell
+# Build container image from local filesystem with Node version as build arg
+docker build --build-arg NODE_VERSION=19 --no-cache -t website:<tag> -f Containerfile .
```

Comment:
```suggestion
# Build container image from local filesystem with default Node version
docker build --no-cache -t website:<tag> -f Containerfile .

# Build container image from local filesystem with Node version as build arg
docker build --build-arg NODE_VERSION=19 --no-cache -t website:<tag> -f Containerfile .
```

This would make sense when we put a default value to in the `Containerfile`.